kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,3 +1,14 @@
+local function check_status(status)
+  if not status then
```

Comment:
This should be unnecessary. If there is no `status`, the `func` check will simply not run.